### PR TITLE
Nginx to serve at /r/

### PR DESCRIPTION
**Pull request by alfredodeza**
It also changes the `/projects` endpoint to `/binaries/` 

Sorry for the feature creep.

### CODE REVIEW

File: chacra/tests/controllers/repos/test_repos.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def test_repo_exists(self, session):
         )
         repo.path = "some_path"
         session.commit()
-        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/api/')
+        result = session.app.get('/repos/foobar/firefly/ubuntu/trusty/')
```
Comment (andrewschoen) [inline chacra/tests/controllers/repos/test_repos.py]: Why did you need to change this? Did this change break the /api endpoint or do you plan for that endpoint to no longer be needed because of nginx serving everything at /r/?

Comment (alfredodeza) [inline chacra/tests/controllers/repos/test_repos.py]: I was thinking that we wanted `api/` to provide something aside from the normal repo directories, but since that is no longer the case because the actual repo will live elsewhere, then I took it out. Hope that is OK?

Comment (andrewschoen) [inline chacra/tests/controllers/repos/test_repos.py]: That's fine, I just wanted to confirm the intent.  The three controller methods for the ``api/`` endpoint still exist in ``controllers/repos/__init__.py`` though. We should probably remove those to avoid confusion.
